packages/@okta/vuepress-site/docs/concepts/policies/index.md

```diff
@@ -4,116 +4,151 @@ meta:
 
-**Control which application can access what information from your APIs**
+You can create an authentication policy specifically for the app or create a few policies and [share them](https://help.okta.com/okta_help.htm?type=oie&id=ext-share-auth-policy) across multiple apps.
 
-When you want to restrict access to an API based on the calling application, you can create an [access policy](/docs/reference/api/authorization-servers/#policy-object) to do that. Access policies are also good when you need scopes in addition to the [reserved scopes](/docs/reference/api/oidc/#scopes) that are created with any Okta authorization server. For example, when you want to improve compatibility for an application, you can return additional profile information for the user by [creating custom scopes with corresponding claims](/docs/guides/customize-authz-server/main/#create-scopes) that tie them to a piece of user information.
+Use the [Authentication Policies page](https://help.okta.com/okta_help.htm?type=oie&id=ext-create-auth-policy) to modify an app's sign-on policy or switch to a different policy. See [Configure a global session policy and an authentication policy](/docs/guides/configure-signon-policy/main/). Also, you can [merge duplicate authentication policies with identical rules](https://help.okta.com/okta_help.htm?type=oie&id=ext-merge-auth-policies) to simplify policy management.
 
-### How policies work
+### Password policies
 
-Policy settings for a particular policy type consist of one or more Policy objects, each of which contains one or more policy rules. Policies contain groups of resources that require similar treatment, such as apps with the same security characteristics or user groups with the same account setup requirements. Rules describe the conditions of policy behavior, such as requests from a geographical location or whether the user is on or off a trusted network. Okta combines the conditions of a policy and the conditions of a rule to determine whether a policy is applied.
+A [password policy](/docs/reference/api/policy/#password-policy) determines the requirements for a user's password length and complexity, and how often a user must change their password. Okta provides a default policy to enforce the use of strong passwords.
 
-As a best practice, restrictive rules should be placed at the top of the priority list. Beyond that, you can create combinations of conditions for multiple scenarios. There is no limit to the number of rules that your policies can have.
+See [Configure the password authenticator](https://help.okta.com/oie/en-us/content/topics/identity-engine/authenticators/configure-password.htm) for more information on passwords as an authenticator factor.
```

Comment:
(https://help.okta.com/oie/en-us/content/topics/identity-engine/authenticators/configure-password.htm) needs to be an alias

```suggestion
See [Configure the password authenticator](https://help.okta.com/oie/en-us/content/topics/identity-engine/authenticators/configure-password.htm) for more information on passwords as an authenticator factor.
```